coremltools/converters/mil/frontend/torch/ops.py

```diff
@@ -6447,15 +6448,20 @@ def scaled_dot_product_attention(context, node):
     See details at:
     https://pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html
     """
-    inputs = _get_inputs(context, node, min_expected=3)
-    q, k, v = inputs[: 3]
-    attn_mask = None if len(inputs) < 4 else inputs[3]
-    dropout = 0.0 if len(inputs) < 5 else inputs[4]
-    is_causal = False if len(inputs) < 6 else inputs[5].val
-    if attn_mask is not None and is_causal:
+    inputs = _get_inputs(context, node, expected=[6, 7])
```

Comment:
We should keep the current (less strict requirement) of `min_expected=3` rather than `expected=[6, 7]`. We should keep the logic from the above deleted code that set those to default values if they are not present.

Reply:
Done!